float32 maxC
float32 minC
---
bool ack